PL/Moodles_PL.txt

```diff
@@ -0,0 +1,114 @@
+  Moodles_pain_desc_lvl2 = "20%^ Bólu<br>ograniczona widoczność w trybie zbieractwa<br>+10% szansa na potknięcie się podczas biegu i skakania<br>-15% obrażeń ataku<br>-15% szybkość ataku<br>-15% prędkość poruszania się<br>niemożliwe do spania",
+  Moodles_pain_desc_lvl3 = "50%^ Bólu<br>znacznie ograniczona widoczność w trybie zbieractwa<br>+15% szansa na potknięcie się podczas biegu i skakania<br>-20% obrażeń ataku<br>-15% szybkość ataku<br>-15% prędkość poruszania się<br>niemożliwe do spania",
+  Moodles_pain_desc_lvl4 = "75%^ Bólu<br>straszliwie ograniczona widoczność w trybie zbieractwa<br>+20% szansa na potknięcie się podczas biegu i skakania<br>-35% obrażeń ataku<br>-15% szybkość ataku<br>-15% prędkość poruszania się<br>niemożliwe do spania<br>niemożliwe do ćwiczeń",
+  -- Bleeding
+  Moodles_bleed_desc_lvl1 = "jedna część twojego ciała krwawi<br>wolno tracisz punkty zdrowia",
+  Moodles_bleed_desc_lvl2 = "dwie części twojego ciała krwawią<br>-1 punkt pojemności ekwipunku<br>tracisz punkty zdrowia",
+  Moodles_bleed_desc_lvl3 = "trzy części twojego ciała krwawią<br>-1 punkt pojemności ekwipunku<br>szybko tracisz punkty zdrowia",
+  Moodles_bleed_desc_lvl4 = "cztery lub więcej części twojego ciała krwawią<br>-1 punkt pojemności ekwipunku<br>bardzo szybko tracisz punkty zdrowia",
+  -- Hyperthermia
+  Moodles_hyperthermia_desc_lvl1 = "temperatura 37,5°C<br> szybciej się męczysz<br>pragnienie zwiększone",
+  Moodles_hyperthermia_desc_lvl2 = "temperatura 39°C<br>szybko się męczysz<br>pragnienie znacznie się zwiększa<br>-34% szybkości ataku<br>-34% prędkości poruszania",
+  Moodles_hyperthermia_desc_lvl3 = "temperatura 40°C<br>bardzo szybko się męczysz<br>pragnienie się bardzo zwiększa<br>-67% szybkości ataku<br>-67% prędkości poruszania",
+  Moodles_hyperthermia_desc_lvl4 = "temperatura 41°C<br>szybko tracisz siły<br>pragnienie bardzo szybko się zwiększa<br>-90% szybkości ataku<br>-100% prędkości poruszania<br>szybka utrata HP",
+  -- Hypothermia
+  Moodles_hypothermia_desc_lvl1 = "temperatura 36,5°C<br>brak efektów",
+  Moodles_hypothermia_desc_lvl2 = "temperatura 35°C<br>-34% szybkości ataku<br>-34% prędkości poruszania<br>niewielkie szanse na przeziębienie",
+  Moodles_hypothermia_desc_lvl3 = "temperatura 30°C<br>-67% szybkości ataku<br>-67% prędkości poruszania<br>słabe szanse na przeziębienie",
+  Moodles_hypothermia_desc_lvl4 = "temperatura 25°C<br>-90% szybkości ataku<br>-100% prędkości poruszania<br>duże szanse na przeziębienie<br>tracisz punkty życia",
+  -- Windchill
+  Moodles_windchill_desc_lvl1 = "Odczuwalna temperatura 5°C niższa niż rzeczywista",
+  Moodles_windchill_desc_lvl2 = "Odczuwalna temperatura 10°C niższa niż rzeczywista",
+  Moodles_windchill_desc_lvl3 = "Odczuwalna temperatura 15°C niższa niż rzeczywista",
+  Moodles_windchill_desc_lvl4 = "Odczuwalna temperatura 20°C niższa niż rzeczywista",  
+	-- Dead
+	Moodles_dead_lvl1 = "Spoczywaj w pokoju.",
+	-- Zombified
+	Moodles_zombie_lvl1 = "Najgorsze zakończenie.",
```

Comment:
Since this is the last line, you don't need to add an `,` at the end of it.

You only need to add an `,` when there's more lines after it.

```suggestion
	Moodles_zombie_lvl1 = "Najgorsze zakończenie."
```

Reply:
Okey its done. :)